gridcomps/FakeDyn/FakeDynGridComp.F90

```diff
@@ -0,0 +1,163 @@
+#include "MAPL_Generic.h"
```

Comment:
Do you really need a new directory for this?    We have gridcomps.

Reply:
Not really. I'll pull it out.

Reply:
On second thought, the separate directory lets me create a separate `libMAPL.fakedyn.so` easily.

Reply:
?    The existing directory does that for the other test gridcomps.   We'll discuss tomorrow.

Reply:
Ah, I had it in the wrong place - in the top level `gridcomps` directory. Moved it to `generic3g/tests/gridcomps`.

Reply:
Hmm - how did I miss that.   It looked like it was just one level too far down ...